collection/Griffin Macaulay; The Griffon's Saddlebag, Book 4.json

```
@@ -14128,64 +14387,661 @@
+						"When you play this note, you can communicate a single word to any plant that hears it. The plants understand the word, although they gain no special ability to respond as a result of using the horn."
+					]
+				},
+				{
+					"type": "entries",
+					"name": "Note of Plant Command (5 charges)",
+					"entries": [
+						"When you play this note, any plant creatures that can hear it must make a DC 15 Wisdom saving throw. On a failed save, a creature is {@condition charmed} by you for 1 hour or until you or any of your allies harm or threaten it. You can issue a command as part of blowing the horn. Any plants {@condition charmed} by the horn immediately obey the command, which they continue to follow until the task is complete, until you issue another command using this property or the Note of Communication, or until they're no longer {@condition charmed}."
+					]
+				},
+				{
+					"type": "entries",
+					"name": "Note of Sun (1 charge)",
+					"entries": [
+						"When you play this note, any fertile soil within the horn's audible range immediately begins to grow a thick layer of grass; flowers within that range turn to face the horn as if it were the sun."
+					]
+				},
+				"When you expend the last charge from the horn, roll a {@dice d20}. On a 1, the horn rapidly wilts, rots, and turns to soil in your hands, destroying it."
+			],
+			"attachedSpells": [
+				"spike growth"
+			],
+			"fluff": {
+				"images": [
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "https://raw.githubusercontent.com/TheGiddyLimit/homebrew-img/main/img/GriffonsSaddlebag4/Items/Horn-of-Verdance.webp"
+						}
+					}
+				]
+			}
+		},
+		{
+			"name": "Prismatic Mirror",
+			"source": "GriffonsSaddlebag4",
+			"page": 0,
+			"baseItem": "shield|phb",
+			"type": "S",
+			"rarity": "rare",
+			"reqAttune": true,
+			"weight": 6,
+			"ac": 2,
+			"bonusAc": "+1",
+			"entries": [
+				"This shield is a hand-carved wooden frame with a perfect, silver mirror set inside it. Despite being flat, the mirror refracts light as if it were curved, like a heater shield. While holding this shield, you have a +1 bonus to AC. This bonus is in addition to the shield's normal bonus to AC.",
+				{
+					"type": "entries",
+					"name": "Reflect Light",
+					"entries": [
+						"Whenever you take radiant damage while holding the shield, you can use your reaction to halve the radiant damage you take. When you do, choose a target that you can see within 60 feet of you that can see you. That target must make a DC 15 Constitution saving throw or take damage equal to the amount of radiant damage you reduced. Roll a {@dice d4} and use the table below to determine the type of damage dealt and its additional effects; each result corresponds to a color of light that's reflected at the target."
+					]
+				},
+				{
+					"type": "table",
+					"colLabels": [
+						"d4",
+						"Light Color",
+						"Damage",
+						"Type\tEffect"
```

```suggestion
						"Type/Effect"
```
 If you want a backslash you have to escape it with an extra backslash as you're inside a json file.